src/main/java/ch/wisv/areafiftylan/controller/ConsumptionController.java

```diff
@@ -0,0 +1,81 @@
+package ch.wisv.areafiftylan.controller;
+
+import ch.wisv.areafiftylan.dto.ConsumptionDTO;
+import ch.wisv.areafiftylan.exception.AlreadyConsumedException;
+import ch.wisv.areafiftylan.exception.ConsumptionNotFoundException;
+import ch.wisv.areafiftylan.model.util.Consumption;
+import ch.wisv.areafiftylan.service.ConsumptionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+import static ch.wisv.areafiftylan.util.ResponseEntityBuilder.createResponseEntity;
+
+/**
+ * Created by beer on 16-5-16.
+ */
+@RestController
+@PreAuthorize("hasRole('ADMIN')")
+@RequestMapping(value = "/consumptions")
+public class ConsumptionController {
+    @Autowired
+    private ConsumptionService consumptionService;
+
+    @RequestMapping(method = RequestMethod.GET)
+    public Map<String, Boolean> isConsumed(@RequestBody ConsumptionDTO consumptionDTO){
```

Comment:
I think a `@RequestBody` for a `GET` request is a bit weird. I would turn this into `/consumptions/{consumptionId}/tickets/{ticketId}`. More RESTful too